Gems/Atom/RPI/Code/Include/Atom/RPI.Edit/Common/JsonUtils.h

```diff
@@ -119,6 +123,62 @@ namespace AZ
                 return false;
             }
 
+            using DeserializedCpuData = AZStd::vector<RHI::CpuProfilingStatisticsSerializer::CpuProfilingStatisticsSerializerEntry>;
+            inline Outcome<DeserializedCpuData, AZStd::string> LoadSavedCpuProfilingStatistics(const AZStd::string& capturePath)
```

Comment:
This is a big function to declare as inline. What's the reasoning behind this?

Reply:
I had initially done this since the rest of the files in the header were templates, meaning implicitly inline - this one was sort of an outlier since it's the only non-templated function (and the `inline` was to avoid multiple definitions). You're right though, that's kind of a hack and this should probably be moved to the .cpp (which is empty as of now strangely enough).

Reply:
Done, had to change `CMakeLists.txt` since the linker was complaining about not finding a definition for the function.